mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/job/JobActor.java

```diff
@@ -1286,6 +1288,8 @@ class WorkerManager implements IWorkerManager {
                 .build();
         private volatile boolean stageAssignmentPotentiallyChanged;
 
+        private WorkerManagerLogger workerLogger;
```

Comment:
Introducing extra lifecycle for these loggers doesn't seem to provide enough benefits. Adding the extra metadata to the logger message is sufficient enough.

Reply:
Hi, @Andyz26 can you give me some hints, like what metadata I could add and how.

Reply:
The issue is about jobId at the job actor level and that is a different structure comparing to this workerManager. Here WorkerManager is not 1 to 1 mapping with a specific worker logically. There is no need to maintain a worker-level logger as long as the information needed is logged.

Reply:
so you mean, I can directly add worker Id to Logger.info, where ever worker id is not mentioned.